layouts/shortcodes/observability_pipelines/processors/filter_syntax.md

```diff
@@ -11,6 +11,7 @@ Filter query examples:
 - `status:ok service:flask-web-app`: This filters for all logs with the status `OK` from your `flask-web-app` service.
     - This query can also be written as: `status:ok AND service:flask-web-app`.
 - `host:COMP-A9JNGYK OR host:COMP-J58KAS`: This filter query only matches logs from the labeled hosts.
+- `@user.status:inactive`: This filters for logs with the key-value attribute `status:inactive` nested under `user`.
```

Comment:
What is user here? Is it a tag? If so I'd say "nested under the `user` tag", for example

Reply:
Should be the `user` attribute. Updated!